printer/json.go

```diff
@@ -11,17 +11,18 @@ import (
 
 type JsonPrinter struct {
 	DarkBackground bool
+	ColorSchema    ColorSchema
```

Comment:
Really nice that you're passing around the schema instead of using globals. :)

Reply:
yep, and actually we could remove the `DarkBackground` from a lot of places and the dark/light option should be checked at start, only if no custom color schema is provided. 
I kept it til I have something working as a demo... which is the case now. now it's cleanup time.
The `ColorSchema` definition is maybe not defined at the right place either (in `printer` for now)...